### DISCUSSION TITLE
trino mysql connector type mapping question

### DISCUSSION THREAD

**Discussion by Firstero** [trinodb/trino#18054] [Q&A]
"I created a MySQL table using Trino, but the Trino varchar data type was mapped to MySQL's longtext data type, which is inconsistent with the documentation."

1. In trino , describe origin_table result as follow

![image](https://github.com/trinodb/trino/assets/68093523/c480b6e6-13f7-4939-a5a0-03a6545f398c)
2. create table as select * from origin_table
3. In trino describe target_table result  as follow
![image](https://github.com/trinodb/trino/assets/68093523/fdeb6ea1-b764-4422-b2fc-512488978ac1)
4. but in mysql client, describe target_table result as follow
![image](https://github.com/trinodb/trino/assets/68093523/ad4d4042-fec3-493a-8243-24f1072a44cb)

**Comment by Firstero** [comment]
trino version 405, mysql server 5.7.26

**Comment by ebyhr** [comment]
We should fix the documentation. 
cc: @mosabua

**Comment by mosabua** [comment]
@ryandoowan please send a PR with that fix. Get help from @colebow or myself as needed.
